darwin/importer/importer.py

```diff
@@ -1308,6 +1309,10 @@ def _import_annotations(
 
         if annotation.id:
             serial_obj["id"] = annotation.id
+        else:
+            annotation_id = str(uuid.uuid4())
+            annotation.id = annotation_id
+            serial_obj["id"] = annotation_id
```

Comment:
simplifying this one:

```suggestion
        annotation.id = annotation.id or str(uuid.uuid4())
        serial_obj["id"] = annotation.id
```